package.json

```diff
@@ -1,18 +1,31 @@
 {
     "name": "decimal128",
     "version": "12.2.0",
-    "main": "src/decimal128.js",
+    "main": "dist/cjs/decimal128.cjs",
+    "module": "dist/esm/decimal128.mjs",
+    "typings": "dist/esm/decimal128.d.ts",
+    "exports": {
+        ".": {
+            "import": "./dist/esm/decimal128.mjs",
+            "require": "./dist/cjs/decimal128.cjs",
```

Comment:
check out https://github.com/nodejs/node/issues/52174, this pattern is not ideal and we should use the alternative mentioned in the thread.